### PR TITLE
Made a new class for inbox related test case. This class covers scena…

### PR DESCRIPTION
**Pull request by PriyaJain5**
Made a new class for inbox related test case. This class covers scenario for NMS_FT_108, NMS_FT_121

### CODE REVIEW

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

**Comment by larubbio** [comment]
add to whitelist

**Comment by frankhuster** [comment]
ok to test

**Comment by frankhuster** [comment]
ok to test

File: testing/src/test/java/org/motechproject/nms/testing/it/kilkari/InboxServiceBundleIT.java [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+				SubscriptionPackBuilder.createSubscriptionPack(
+						"pregnancyPack",
+						SubscriptionPackType.PREGNANCY, 			
+						SubscriptionPackBuilder.PREGNANCY_PACK_WEEKS,	//72
+						2));											// messages per week
+
+		pregnancyPack = subscriptionPackDataService.byName("pregnancyPack"); 
+	}
+
+	private void cleanupData() {
```
Comment (frankhuster) [inline testing/src/test/java/org/motechproject/nms/testing/it/kilkari/InboxServiceBundleIT.java]: When creating new code (or updating existing code) please use TestingService.clearDatabase() to delete the database. That will reduce code duplication & potential errors.

**Comment by frankhuster** [comment]
ok to test

**Comment by frankhuster** [comment]
@PriyaJain5, thanks for the work, looks great. I merged master & fixed up the code so it would use the new IT framework (that Rob & I changed over the weekend). Can you please change `verifyFT108_121()` to be two methods, one for each FT, and make sure to include the text description for each individual FT?

Thanks!

**Pull request merged**